apps/test/unit/templates/teacherDashboard/SectionActionDropdownTest.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import {shallow} from 'enzyme';
 import {expect} from '../../../util/configuredChai';
```

Comment:
should we move this to reconfiguredChai too?

Reply:
same for the above file

Reply:
@jmkulwik Can you clarify this? I've received mixed messages: I heard we want to move to reconfiguredChai (and drop chai-enzyme) but then I later heard that doesn't matter.  For what it's worth, these tests passed with the enzyme 3 upgrade in place.

Reply:
@islemaster, We do want to move to reconfiguredChai because chai-enzyme isn't being actively maintained anymore. 

There was some confusion because we had thought we needed to remove chai-enzyme to complete the React upgrade, but it turns out that is not the case. If we aren't finished moving to reconfiguredChai by the time we are ready for the React upgrade, we can temporarily move chai-enzyme to a year-old beta version (1.0.0-beta.1).

Reply:
I remember you telling me that now, sorry for asking the same question twice! Got it.  I'm going to consider that an orthogonal concern to this particular PR since I've got these tests passing under both configurations, but will keep it in mind for future updates.  Thanks!